client/packages/common/src/intl/locales/en/common.json

```diff
@@ -85,6 +87,7 @@
   "error.no-service-charges": "There are no service charges assigned to this store",
   "error.no-temperature-breaches": "No temperature breaches found",
   "error.problem-saving": "An error occurred: there was a problem saving the data.",
+  "error.an-error-occurred": "An error occurred: {{message}}",
```

Comment:
Might be good to let the team know about this one, to avoid accidental similar translations being created?
I like the idea of this...